[plugin] flake8_pytest_style without config section (pyproject.toml) crashes

minimal reproducing config:
```toml
select = ["PT"]
```

output:
```
[2023-01-03][13:28:56][ruff::cache][ERROR] Failed to deserialize encoded cache entry: Io(Kind(UnexpectedEof))
[2023-01-03][13:28:56][ruff::cache][ERROR] Failed to deserialize encoded cache entry: Io(Kind(UnexpectedEof))
thread '<unnamed>' panicked at 'called `Option::unwrap()` on a `None` value', src/flake8_pytest_style/plugins/parametrize.rs:196:29
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

--- a/src/flake8_pytest_style/plugins/parametrize.rs
+++ b/src/flake8_pytest_style/plugins/parametrize.rs
@@ -193,13 +193,14 @@ pub fn parametrize(checker: &mut Checker, decorators: &[Expr]) {
     if let Some(decorator) = decorator {
         if let ExprKind::Call { args, .. } = &decorator.node {
             if checker.settings.enabled.contains(&CheckCode::PT006) {
-                let first = args.first().unwrap();
-                check_names(checker, first);
+                if let Some(arg) = args.get(0) {
+                    check_names(checker, arg);
+                }
             }
-
             if checker.settings.enabled.contains(&CheckCode::PT007) {
-                let second = args.get(1).unwrap();
-                check_values(checker, second);
+                if let Some(arg) = args.get(1) {
+                    check_values(checker, arg);
+                }
             }
         }
     }

